ui/plugin-system/src/components/LegendOptionsEditor/LegendOptionsEditor.tsx

```diff
@@ -63,9 +63,10 @@ const VALUE_OPTIONS: LegendValueOption[] = Object.entries(LEGEND_VALUE_CONFIG).m
 export interface LegendOptionsEditorProps {
   value?: LegendSpecOptions;
   onChange: (legend?: LegendSpecOptions) => void;
+  includeLegendCalcs?: boolean;
```

Comment:
Not sure about the naming, I don't get the "Calcs", maybe I missing something? Well, the naming "Value" is not helping either, I don't know why it has not been called Sorting :thinking: